src/bionics.cpp

```diff
@@ -971,13 +975,14 @@ bool player::install_bionics( const itype &type, int skill_level )
 
     const int pk = get_painkiller();
     int pain_cap = 100;
-    if( has_trait( trait_id( "PAINRESIST_TROGLO" ) ) ) {
+    if( has_trait( "PAINRESIST_TROGLO" ) ) {
```

Comment:
You want to use the constants above, like this:
`has_trait( trait_PAINRESIST_TROGLO )`